scripts/sfpshow

```diff
@@ -262,7 +264,12 @@ def display_invalid_intf_status(intf_name):
     click.echo(output)
 
 class SFPShow(object):
-    def __init__(self, intf_name, namespace_option, dump_dom=False):
+    PM_WINDOW_NUM_IN_DB = {'60sec': 15, '15min': 12, '24hrs': 2}
```

Comment:
Please define constants for start number/number of windows of 60sec/15min/24hrs window and use

Reply:
this PM_WINDOW_NUM_IN_DB/PM_WINDOW_START_INDEX_IN_DB is already constants (just in the form of dict) for start number/number of windows of 60sec/15min/24hrs window